core/lib/server/_legacy-integration-queries.ts

```diff
@@ -83,7 +87,8 @@ export const _getPubType = async (pubTypeId: string): Promise<GetPubTypeResponse
 	if (!pubType) {
 		throw new NotFoundError("Pub Type not found");
 	}
-	return pubType;
+	const { fields, ...rest } = pubType;
+	return { ...rest, fields: fields.map((f) => f.pubField) };
```

Comment:
I'm not great with prisma and wasn't sure how to get the query to not nest `pubField` there—it seems like now that I've explicitly defined the join table, prisma doesn't want to allow the direct `fields` selection anymore. so this part just rejiggers the result from 

```js
fields: [
  {
    pubField: {
      id: '',
      name: '',
      slug: '',
      schema: {...}
    }
  }
]
```

to

```js
fields: [
  {
    id: '',
    name: '',
    slug: '',
    schema: {...}
  }
]
```
which is what the return type expects